src/Core/Components/List/ListComponentBase.razor.cs

```diff
@@ -128,6 +128,11 @@ protected string? InternalValue
     /// </summary>
     [Parameter]
     public virtual Func<TOption, bool>? OptionSelected { get; set; }
+    /// <summary>
```

Comment:
Can you add a space between the two properties?

Reply:
done.